ui/src/app/pages/datasets/dataset-from/dataset-from.component.ts

```diff
@@ -91,52 +106,59 @@ export class DatasetFromComponent implements OnInit {
     } as Dataset;
 
     this.form = this._formBuilder.group({
-      name: [name, [Validators.required, Validators.pattern('^[a-zA-Z_][a-zA-Z0-9_-]*$')]],
-      agent_policy_id: [agent_policy_id, [Validators.required]],
-      agent_group_id: [agent_group_id, [Validators.required]],
-      sink_ids: [sink_ids, [Validators.minLength(1)]],
-    });
+                                          name: [name, [Validators.required, Validators.pattern(
+                                            '^[a-zA-Z_][a-zA-Z0-9_-]*$')]],
```

Comment:
I often add commentary with both valid and invalid examples of a regex to be easier on the eyes of another developer. WDYT?

Reply:
that's definitely a good practice. done!